src/packages/name-service/utils.ts

```diff
@@ -53,3 +54,28 @@ const bnsResolveName = async (baseUrl: string, blockstackId: string): Promise<ob
     }
     return nameData;
 };
+
+export const getCruxIDByAddress = async (bnsUrl: string, walletClientName: string, address: string) => {
```

Comment:
Let's use all bnsURL's here including the one mentioned in wallet config. If response differs from different endpoint then let's consider all names as a valid response which are common across all endpoints.

Reply:
https://app.asana.com/0/1143646564873514/1145788525258245